src/mod/l4_thruster.c

```diff
@@ -133,41 +173,309 @@ int l4_thr_counter = 0; // counter for received bytes
 
 void l4_thruster_init (void *dummy) {
 
-//	char* testvar = "hello thruster";
-//	SysEvent(MODULE_ID_CLIMBAPP, EVENT_INFO, EID_APP_STRING, testvar, strlen(testvar));
+
+	/// **************** PREPROGRAMM SEQUENCES HERE ****************
+
+
+
+	char *wait_between_stages_str ="5000";
+
+	//Build argv array for execution sequence
+
+		static thr_sequences_t temp_sequence[MAX_EXECUTION_SEQUENCE_DEPTH];
```

Comment:
As you found out, you here need the 'static' because you have to reserve this 'variable' somewhere where it stays untouched (not on the stack!). With the MAX_EXECUTION_SEQUENCE_DEPTH it always takes the maximum memory here. So going on from this solution we have to think further: Where do this sequences come from later in the 'real' climb project. Where are they persisted and how can they be 'loaded'/'unloaded' in 'current Sequence' Executing 'slots'. But this is a discussion for later on.....